src/test/scala/htsjdk/UnitSpec.scala

```diff
@@ -0,0 +1,6 @@
+package htsjdk
+
+import org.scalatest.{FlatSpec, Matchers}
+
+/** Base class for all Scala tests. */
+class UnitSpec extends FlatSpec with Matchers
```

Comment:
I see. OK. thanks. I was confused since you used Htsjdk in the baseclass of the java tests